connectors/sources/atlassian.py

```diff
@@ -135,7 +152,7 @@ async def user_access_control_doc(self, user):
                     ACCESS_CONTROL: [<prefixed_account_id>, <prefixed_group_ids>, <prefixed_role_keys>]
                 }
         """
-        account_id = user.get("accountId")
+        account_id = user.get("accountId") or user.get("name")
```

Comment:
Is `user.get("name")` guaranteed to be unique? Looks like `account_id` ends up being used as the access control document's `_id`. 

Is this added because `accountId` is sometimes missing?

Reply:
> Is user.get("name") guaranteed to be unique?

Yes, it is unique for all users.

> Is this added because accountId is sometimes missing?

Exactly, `accountId` is missing while fetching users for non-cloud.